index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile version skeleton</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="navbar">
+  <nav class="container">
+    <a  class="logo" href="#">My logo</a>
+    <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+  </nav>
+</header>
+<section class="hero">
+  <div class="container">
+  <h2 class="primary-text">I’m Asiati Magana <br> Glade to see you!</h2>
```

Comment:
Kindly replace `<h2>` with `<h1>` tag and make sure there's only one `<h1>` tag on a web page for SEO reasons.